packages/components/table/src/util.ts

```diff
@@ -411,16 +423,7 @@ export function createTablePopper(
       },
     })
   }
-  const popperOptions = tooltipOptions.popperOptions || {}
-  popperInstance = createPopper(trigger, content, {
-    placement: tooltipOptions.placement || 'top',
-    strategy: 'fixed',
-    ...popperOptions,
-    modifiers: popperOptions.modifiers
-      ? modifiers.concat(popperOptions.modifiers)
-      : modifiers,
-  })
-  trigger.addEventListener('mouseenter', onOpen)
+  trigger.addEventListener('mouseenter', onOpen, true)
```

Comment:
Why should `useCapture` be set for the event binding here?

Reply:
Because the onOpen event was not triggered during the event bubbling process, it can be triggered during the event capturing process